js/index.js

```diff
@@ -40,3 +40,110 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+// ## Task 1: Create selectors to point your data into elements
+// * [ ] Create selectors by using any of the DOM element's methods
+// * [ ] Note that IDs have been used on all images.  Use the IDs to update src path content
+
+//  ## Task 2: Update the HTML with the JSON data
+// * [ ] Remember, NO direct updating of the HTML source is allowed.
+// * [ ] Using your selectors, update the content to match the example file.
+// * [ ] Remember to update the src attributes on images
+
+const nav = document.querySelectorAll('a');
+nav[0].textContent = siteContent ["nav"] ["nav-item-1"];
+nav[1].textContent = siteContent ["nav"] ["nav-item-2"];
+nav[2].textContent = siteContent ["nav"] ["nav-item-3"];
+nav[3].textContent = siteContent ["nav"] ["nav-item-4"];
+nav[4].textContent = siteContent ["nav"] ["nav-item-5"];
+nav[5].textContent = siteContent ["nav"] ["nav-item-6"];
+
+
+const textHeading = document.querySelectorAll('h4');
+textHeading[0].textContent = siteContent ["main-content"] ["features-h4"];
+textHeading[1].textContent = siteContent ["main-content"] ["about-h4"];
+textHeading[2].textContent = siteContent ["main-content"] ["services-h4"];
+textHeading[3].textContent = siteContent ["main-content"] ["product-h4"];
+textHeading[4].textContent = siteContent ["main-content"] ["vision-h4"];
+
+const pContent = document.querySelectorAll('p');
+pContent[0].textContent = siteContent ["main-content"] ["features-content"];
+pContent[1].textContent = siteContent ["main-content"] ["about-content"];
+pContent[2].textContent = siteContent ["main-content"] ["services-content"];
+pContent[3].textContent = siteContent ["main-content"] ["product-content"];
+pContent[4].textContent = siteContent ["main-content"] ["vision-content"];
```

Comment:
I like the organization here. It makes it easy to see what you're capturing. Comments above the code about what section or function this serves would take it to the next level